src/plat/bcm2837/overlay-rpi3.dts

```diff
@@ -20,4 +20,15 @@
 		/* This is configurable in the Pi's config.txt, but we use 128MiB of RAM by default. */
 		reg = <0x00 0x08000000>;
 	};
+
+	reserved-memory {
+		#address-cells = < 0x01 >;
+		#size-cells = < 0x01 >;
+		ranges = < >;
+/* Memory page holding the bootloader armstub and secondary core parking spinloops */
```

Comment:
I think this comment should be much more verbose explaining all the things you also mentioned in this PR. Please note that the PR and commit comment may not be available to somebody looking at this code in the future. And the importance of this when *not* building for SMP is not this obvious.

Reply:
Added a more in depth explanation to the comment. Do you think this take covers the issue good enough or did I miss any important aspect?